src/autolabel/labeler.py

```diff
@@ -170,17 +170,12 @@ def run(
 
         indices = range(current_index, len(dataset.inputs))
 
-        if self.console:
```

Comment:
We need the ability to not print to the console. @nihit had earlier removed this for use in the cloud product, want to confirm with him if we should remvoe this

Reply:
I didn't remove the feature, rich has a built in parameter to pass to the console when creating it called `quiet` which suppresses all output. I set this to `not console_output` in the constructor instead of using if statements everywhere since I think it's a cleaner solution (and also resolves the kill signal bug).